hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/FrameRunner.java

```diff
@@ -167,6 +169,8 @@ private long effectiveGasUsed(final long gasLimit, @NonNull final MessageFrame f
     private void propagateHaltException(MessageFrame frame, ExceptionalHaltReason haltReason) {
         if (haltReason.equals(INSUFFICIENT_CHILD_RECORDS)) {
             setPropagatedCallFailure(frame, RESULT_CANNOT_BE_EXTERNALIZED);
+        } else if (haltReason.equals(INVALID_FEE_SUBMITTED)) {
+            setPropagatedCallFailure(frame, HevmPropagatedCallFailure.INVALID_FEE_SUBMITTED);
```

Comment:
L171 and L173 should either _both_ have the `HevmPropagatedCallFailure.` qualification, or _neither_.  (I prefer both.)